backend/src/test/java/com/ambacam/rest/operateurs/OperateursResourceIT.java

```diff
@@ -0,0 +1,413 @@
+		create.setCreatorId(random.nextLong());
+		create.setPaysId(pays1.getId());
+
+		given().contentType(ContentType.JSON).body(create).log().body().post(ApiConstants.OPERATEUR_COLLECTION).then()
+				.log().body().statusCode(400);
+	}
+
+	@Test
+	public void list() {
+		given().get(ApiConstants.OPERATEUR_COLLECTION).then().log().body().statusCode(200)
+				.body("size()", is(equalTo(4))).body("id", containsInAnyOrder(operateur1.getId().intValue(),
+						operateur2.getId().intValue(), creator1.getId().intValue(), creator2.getId().intValue()));
+	}
+
+	@Test
+	public void get() {
+		given().get(ApiConstants.OPERATEUR_ITEM, operateur2.getId()).then().log().body().statusCode(200)
+				.body("id", is(equalTo(operateur2.getId().intValue()))).body("nom", is(equalTo(operateur2.getNom())))
+				.body("prenom", is(equalTo(operateur2.getPrenom()))).body("sexe", is(equalTo(operateur2.getSexe())))
+				.body("nationalite.id", is(equalTo(operateur2.getNationalite().getId().intValue())))
+				.body("creePar.id", is(equalTo(operateur2.getCreePar().getId().intValue())))
+				.body("creeLe", is(equalTo(operateur2.getCreeLe().getTime())));
+	}
+
+	@Test
+	public void getNotFound() {
+		given().get(ApiConstants.OPERATEUR_ITEM, random.nextLong()).then().statusCode(404);
+	}
+
+	@Test
+	public void delete() {
+
+		operateur2.setCreePar(operateur1);
+		repository.save(operateur2);
+
+		given().delete(ApiConstants.OPERATEUR_ITEM, operateur1.getId()).then().statusCode(200);
+
+		// check that the operateur has been deleted
+		Operateur actual = repository.findOne(operateur1.getId());
+		assertThat(actual, is(nullValue()));
+
+		Operateur actualOperateur2 = repository.findOne(operateur2.getId());
+
+		// check unlink with created operateurs
+		assertThat(actualOperateur2.getCreePar(), is(nullValue()));
+	}
+
+	@Test
+	public void deleteNotFound() {
+		given().delete(ApiConstants.OPERATEUR_ITEM, random.nextLong()).then().statusCode(404);
+	}
+
+	@Test
+	public void update() {
+
+		OperateurUpdateTO update = buildOperateurUpdateTO();
+
+		update.setPaysId(pays1.getId());
+
+		given().contentType(ContentType.JSON).body(update).put(ApiConstants.OPERATEUR_ITEM, operateur2.getId()).then()
+				.log().body().statusCode(200);
+
+		// check that the operateur has been saved
+		Operateur actual = repository.findOne(operateur2.getId());
+		assertThat(actual, is(notNullValue()));
+		assertThat(actual.getNom(), is(equalTo(update.getNom())));
+		assertThat(actual.getPrenom(), is(equalTo(update.getPrenom())));
+		assertThat(actual.getSexe(), is(equalTo(update.getSexe())));
+		assertThat(actual.getUsername(), is(equalTo(operateur2.getUsername())));
+		assertThat(actual.getPassword(), is(equalTo(operateur2.getPassword())));
+		assertThat(actual.getNationalite().getId(), is(equalTo(pays1.getId())));
+		assertThat(actual.getCreePar().getId(), is(equalTo(operateur2.getCreePar().getId())));
+		assertThat(actual.getCreeLe().getTime(), is(equalTo(operateur2.getCreeLe().getTime())));
+
```

Comment:
Please check all properties

Reply:
Please can you separate with comments the part of the properties that has been updated and the part that has not been updated?